test/scripts/incentive-spreadsheet-to-json.test.ts

```diff
@@ -127,3 +130,13 @@ test('drop records with omit_from_api marked', tap => {
   tap.equal(validStateIncentives.length, 0);
   tap.end();
 });
+
+test('extract IDs from a standard URL', tap => {
+  const url =
```

Comment:
add test for failure cases, of failed spreadsheet ID, incentive data sheet ID, or not matching docs regex?

Probably best to cover all bases here.

Reply:
Done. I had to reconfigure things and introduce an interface so I could fake the sheets client.

Reply:
Thanks! Tests LGTM.